lntest/itest/lnd_channel_backup_test.go

```diff
@@ -987,6 +987,30 @@ func testChanRestoreScenario(t *harnessTest, net *lntest.NetworkHarness,
 	)
 	require.NoError(t.t, err)
 
+	// We now need to make sure the server is fully started before we can
+	// actually close the channel. This is the first check in CloseChannel
+	// so we can try with a nil channel point until we get the correct error
+	// to find out if Dave is fully started.
+	err = wait.Predicate(func() bool {
+		const expectedErr = "must specify channel point"
+		resp, err := dave.CloseChannel(
+			ctxt, &lnrpc.CloseChannelRequest{},
+		)
+		if err != nil {
+			return false
+		}
+
```

Comment:
should there be a `defer resp.Close()` here?

Reply:
I don't think client stream has a `Close`.

Reply:
Added a `resp.CloseSend()` that's all we can do from the client side (and also cancel the context).

Reply:
had to look this up, there are apparently 4 ways to properly close a stream https://godoc.org/google.golang.org/grpc#ClientConn.NewStream. seems like cancelling the context is actually what we want here

Reply:
but i don't think `CloseSend` hurts anything sooo 🤷‍♂️